src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.util.quantities;
+
+import javax.measure.Quantity;
+import tec.uom.se.ComparableQuantity;
+import tec.uom.se.quantity.Quantities;
+
+/** Offers useful methods to handle {@link Quantity}s */
+public class QuantityUtil {
+  private QuantityUtil() {
+    throw new IllegalStateException("Do not instantiate a utility class.");
+  }
```

Comment:
Please prefer error messages in passive -- "Utility classes cannot be instantiated"